### ISSUE TITLE
Can't inherit readable stream from stream

### ISSUE THREAD

**Issue by you-fail-me** [nodejs/readable-stream#226]
I'm using a package that depends on nodejs core modules like buffer and stream. I used `rn-nodeify` to substitute those dependencies with their client counterparts to make the package able to work on react native, and now when that package requires stream it uses your readable-stream implementation.
But when I try to run the react native app on a device I'm getting an error:

```
Object prototype may only be an Object or null.
inherits
inherits_browser.js:5
<unknown>
_stream_readable.js:52
...
```

As you can see it's unable to inherit readable stream from stream. If I add `console.log` to `_stream_readable.js` I see that after it does `var Stream = require('stream')` variable `Stream` holds an empty object `{}` and it's prototype is undefined so it fails to inherit.
How can I fix this behavior and what is its reason?

**Comment by calvinmetcalf** [comment]
this may be a bug with `rn-nodeify` do you know where the repo for  that is ?

**Comment by you-fail-me** [comment]
this might be a bug of them and I've already asked them to look too. But since the error occurs within your script I thought you could now something that might help me and provide me with a bit of support.

**Comment by calvinmetcalf** [comment]
yeah this is a issue there with [rn-nodeify](https://github.com/mvayngrib/rn-nodeify/blob/9917779ed9a461c6c43ba410b9f0bf6372c6f31a/pkg-hacks.js#L99-L112)

**Comment by calvinmetcalf** [comment]
you may beed to be grabbing events as well as streams as

**Comment by RangerMauve** [comment]
This was probably fixed by #258
